assignments/closure.js

```diff
@@ -1,19 +1,44 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
+let character = 'Ironman'
+function characterSpeak() {
+  return 'I am ' + character + '!';
+}
```

Comment:
Think about what would happen if `character` were inside this function. Would you be able to do `console.log(character)`, outside of the function? Like on line 7?

Right now `character` has global scope, meaning that you could access it from _anywhere_ in your code. What would change if it were confined to the scope of the function `characterSpeak`?